src/en/glossary.md

```diff
@@ -1,26 +1,23 @@
 # Glossary of terms
 
-**Bootstrap** - Initialize a Juju Environment so that Services may be deployed.
+**Bootstrap** - Initialize a Juju environment so that services may be deployed.
```

Comment:
I was debating this too. The terms we're using are tricky. In some contexts a Juju environment is not a unique/proper thingy but in others it is (the Juju Environment). Ditto for "Juju service".